src/components/CoreComponentBadge.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {StyleSheet, View, Text} from 'react-native';
+import {useTheme} from '@react-navigation/native';
+import {SymbolIcon} from 'react-native-xaml';
+
+const createStyles = (colors: any) =>
+  StyleSheet.create({
+    badgeContainer: {
+      height: 28,
+      width: 170,
+      borderRadius: 2,
+      flexDirection: 'row',
+      alignItems: 'center',
+      justifyContent: 'center',
+      backgroundColor: colors.primary,
+      margin: 2,
+    },
+    badgeText: {
+      fontSize: 15,
+      fontWeight: '400',
+      color: 'white',
+      paddingRight: 4,
+    },
+  });
+
+export function CoreComponentBadge() {
+  const {colors} = useTheme();
+  const styles = createStyles(colors);
+  return (
+    <View
+      style={styles.badgeContainer}
+      tooltip="This component is a part of React Native Windows core. No additional library must be added in order to have access to this component in a React Native Windows project.">
+      <Text style={styles.badgeText}>Core Component</Text>
+      <SymbolIcon symbol={57611} foreground="white" />
```

Comment:
Should this be `colors.text`?

Reply:
colors.text is set to a charcoal gray in light theme and a light gray in dark theme, but colors.primary - the button's background - stays constant across themes as blue. I've just manually assigned the text color here so that the text remains visible and constant across theme changes for this badge.